hack/test/e2e-docker.sh

```diff
@@ -25,15 +16,19 @@ function create_cluster {
     --mtu 1500 \
     --memory 2048 \
     --cpus 4.0 \
-    --endpoint "${ENDPOINT}" \
     --with-init-node=false \
     --crashdump
 
   "${TALOSCTL}" config node 10.5.0.2
 }
 
+function destroy_cluster() {
+  "${TALOSCTL}" cluster destroy --name "${CLUSTER_NAME}"
+}
+
```

Comment:
This was required due to some port binding collisions, but I think we can/will fix that outside of this, but still good to destroy so that we can test it.